app/controllers/entities_controller.rb

```diff
@@ -0,0 +1,32 @@
+class EntitiesController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def new
+    @entity = Entity.new
+  end
+
+  def create
+    @entity = current_user.entities.new(entity_params)
+
+    if @entity.save
+      @group_entity = @entity.group_entities.create(group_entity_params)
+      redirect_to group_path(@group_entity.group_id), notice: 'Transaction successful'
+    else
+      flash[:alert] = @entity.errors.full_messages.first
+      render :new
+    end
+  end
+
```

Comment:
- [ ] It will be great to have a list of transactions presented ordered by the most recent. This is a requirement for the transactions page.

Reply:
Requested changes completed.

> Please note that the changes were made in the `GroupsController`, instead of the `EntitiesController`. Also, note that the changes won't be reflected in the video presentation but the changed files.

Reply:
This issue is well resolved! ✔️